cmds/exp/dumpmemmap/main_linux.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Command dumpmemmap prints different kernel interpretations of physical
+// memory address space.
+//
+// Support for:
+//
+//   - /proc/iomem (exists on all systems)
+//   - /sys/firmware/memmap (exists on EFI systems)
+//   - /sys/kernel/debug/memblock (exists on systems with CONFIG_ARCH_KEEP_MEMBLOCK, in particular arm64)
+//   - /sys/firmware/fdt (exists on systems with device trees)
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/dustin/go-humanize"
+	"github.com/u-root/u-root/pkg/boot/kexec"
+	"github.com/u-root/u-root/pkg/dt"
+)
+
+func printMM(mm kexec.MemoryMap) {
+	for _, r := range mm {
+		fmt.Println(" ", r, " ", humanize.Bytes(uint64(r.Range.Size)))
+	}
+}
+
+func main() {
```

Comment:
do you want to make this in the form of run() and then main() just calls run()?

Reply:
I’ll have test coverage for this in a later PR - No need to create indirection yet